portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// When the user scrolls down 80px from the top of the document, resize the navbar's padding and the logo's font size
```

Comment:
This feels like typically it would be in a different PR. It feels like you've got:

1) User Experience
2) Mail impl

as two seperate PRs. You don't need to make any changes here and I totally get it because I make PRs like this as well. But in general I'd consider trying to split features up a bit more. Imagine in a production system if there was a problem with a PR you'd have to roll back (ie Undo) everything not just the problematic part you know?

You don't need to change anything here but consider this for future PRs.